### PR TITLE
#120: API-core-policies-NestedPolicies from md to rst

### PR DESCRIPTION
**Pull request by antoinemeyer5**
### THIS PR:
* API/core/policies/NestedPolicies transition from .md to .rst

- [x] uses cppkokkos
- [x] rebased on latest main 

before (.md) | after (.rst) 
:----------------: | :-------------:
before_1 | image
before_2 | after_2
before_3 | after_3

### CODE REVIEW

File: docs/source/API/core/policies/NestedPolicies.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    Impl::ThreadVectorRangeBoundariesStruct ThreadVectorRange(TeamMemberType team, IndexType begin, IndexType end);
+    Impl::ThreadSingleStruct PerTeam(TeamMemberType team);
+    Impl::VectorSingleStruct PerThread(TeamMemberType team);
+
+Description
+~~~~~~~~~~~
+
+.. cpp:function:: Impl::TeamThreadRangeBoundariesStruct TeamThreadRange(TeamMemberType team, IndexType count);
+
+* Splits the index range ``0`` to ``count-1`` over the threads of the team. This call is potentially a synchronization point for the team, and thus must meet the requirements of ``team_barrier``.
```
Comment (fnrizzi) [inline docs/source/API/core/policies/NestedPolicies.rst:62]: this needs to be indented

**Comment by fnrizzi** [comment]
the description of each function should be indented so that it shows as part of the function

File: docs/source/API/core/policies/NestedPolicies.rst (hunk 1 above)
Comment (antoinemeyer5) [reply docs/source/API/core/policies/NestedPolicies.rst]: fix ✅ my mistake, please excuse me

**Comment by antoinemeyer5** [comment]
indentation corrected, screenshots are up to date ! @fnrizzi & @ajpowelsnl

**Review by fnrizzi — APPROVED** [review]

File: docs/source/API/core/policies/NestedPolicies.rst [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+``Kokkos::ThreadVectorMDRange``
+-------------------------------
+
+Header File: ``<Kokkos_Core.hpp>``
+
+Usage
+~~~~~
+
+.. code-block:: cppkokkos
```
Comment (fnrizzi) [inline docs/source/API/core/policies/NestedPolicies.rst:36]: code block cannot use cppkokkos, a code block is not a sphinx thing it is a rst thing.
Please use 
```
.. code-block:: cpp
```

File: docs/source/API/core/policies/NestedPolicies.rst [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    parallel_for(ThreadVectorRange(team,begin,end), [=] (int i) {});
+    single(PerTeam(team), [=] () {});
+    single(PerThread(team), [=] () {});
+
+Nested policies can be used for nested parallel patterns. In contrast to global policies, the public interface for nested policies is implemented as functions, in order to enable implicit templating on the execution space type via the team handle. 
+
+Synopsis
+~~~~~~~~
+
+.. code-block:: cppkokkos
```
Comment (fnrizzi) [inline docs/source/API/core/policies/NestedPolicies.rst:48]: same as aabove

**Review by fnrizzi — APPROVED** [review]

**Pull request merged**